apps/server/src/modules/sharing/uc/share-token.uc.ts

```diff
@@ -163,6 +171,21 @@ export class ShareTokenUC {
 		});
 	}
 
+	private copyColumnBoard(
+		userId: string,
+		originalColumnBoardId: string,
+		courseId: string
+		// copyName?: string // TODO: implement copyName once is supported
+	): Promise<CopyStatus> {
+		// const destinationCourse = await this.courseService.findById(courseId);
+		// TODO should we not check write permission on destination course here?
```

Comment:
Definitely should check if user has write permission on target. And if target already has child of that ID.
E.g. link gets called twice.